osh/prompt.py

```diff
@@ -242,22 +242,18 @@ def _ReplaceBackslashCodes(self, tokens):
 
         return ''.join(ret)
 
-    def EvalPrompt(self, UP_val):
+    def EvalPrompt(self, s):
         # type: (value_t) -> str
```

Comment:
This should be `type: (str) -> str` now, although I wonder if it will cause any other type errors

Reply:
Yeah, I forgot to update the type annotation. I fixed it. It didn't cause any other type errors because I already updated all the other occurrences of the calls of `EvalPrompt` in commit 03cf713b280749558ab33f2dc1b47df464b86b64.